website/docs/r/mq_broker.html.markdown

```diff
@@ -76,6 +77,11 @@ The following arguments are supported:
 * `id` - (Optional) The Configuration ID.
 * `revision` - (Optional) Revision of the Configuration.
 
+#### `encryption_options`
+
+* `kms_key_id` - (Optional) Amazon Resource Name (ARN) of Key Management Service (KMS) Customer Master Key (CMK) to use for encryption at rest. Requires setting `use_aws_owned_key` to `false`. To perform drift detection when AWS managed CMKs or customer managed CMKs are in use, the value must be configured.
```

Comment:
It is not clear what value must be configured. Is it the `use_aws_owned_key` set to false or are you saying that the `kms_key_id` should be set?

Reply:
I'll switch `the value must be configured` to `this value must be configured` -- we're not the best at documenting `Computed: true` attributes and admittedly I was having a hard time wording it.